core/src/main/java/me/aurium/branch/nodes/builders/AloneBuilder.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ *  Branch
+ *  Copyright © 2021 Aurium
+ *
+ *  Branch is free software: you can redistribute it and/or modify
+ *  It under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  Branch is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with Branch. If not, see <https://www.gnu.org/licenses/>
+ *  and navigate to version 3 of the GNU Affero General Public License.
+ *
+ */
+
+package me.aurium.branch.nodes.builders;
+
+/**
+ * yet another marker interface
```

Comment:
You use so many marker interfaces, but it is not clear to me why

Reply:
Ensure that correctly typed classes are used where they are desired - an AloneNode is preferred for a no-arguments over just general CommandNode